server/controller/admin/admin-controller.js

```diff
@@ -0,0 +1,77 @@
+const { compareSync } = require("bcrypt");
+const { sign } = require("jsonwebtoken");
+const {
+  AdminUserDetails,
+} = require("../../models/adminUserDetails/adminUserDetails");
+const encryptPassword = require("../../utils/hashingPassword");
+const { sendResponse } = require("../../helper/responseHelper");
+
+module.exports = {
+  //route:/api/v1/admin/registerAdmin
+  //access:public
+  registerAdmin: async (req, res, next) => {
```

Comment:
> As of now we will not register admin please comment, instead of this can we create a seed js(Will create a new task) to create a admin using command to login for testing.

ok bro, seeding is good way to feed datas. i will comment that.